lib/sqlite.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+const path = require('path')
+const { open } = require('sqlite')
```

Comment:
What's the advantage of using open from sqlite instead of doing `new sqlite3.Database()`? Is it worth the added dependency?

Reply:
`sqlite` is thin wrapper for `sqlite3`-like libraries. The API is [promise-based](https://github.com/kriasoft/node-sqlite#usage) instead of [callback-based](https://github.com/TryGhost/node-sqlite3#usage), and adds the basic [migrations mechanism](https://github.com/kriasoft/node-sqlite#migrations).